indico/modules/events/reminders/models/reminders.py

```diff
@@ -14,6 +14,7 @@
 from indico.modules.events.registration.models.registrations import Registration
 from indico.modules.events.reminders import logger
 from indico.modules.events.reminders.util import make_reminder_email
+from indico.modules.events.serialize import event_to_ical
```

Comment:
hm... `serialize` vs `ical` or `calendar`? Usually the JSON-based serialization logic (marshmallow schemas) is in `schemas.py` so I wonder if there's a decent risk that we ever add non-ical stuff in here - so I'd probably go for something more ical-specific for that filename.